src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -95,5 +127,35 @@ public static IQueryable<Publication> HydratePublicationForPublicationViewModel(
                 .Include(p => p.Methodology)
                 .Include(p => p.Topic);
         }
+
+        private async Task<Either<ActionResult, bool>> ValidateMethodologyCanBeAssigned(
+            UpdatePublicationMethodologyViewModel model)
+        {
+            if (model.MethodologyId != null && model.ExternalMethodology == null)
+            {
+                var methodology = await _context.Methodologies.FirstOrDefaultAsync(p => p.Id == model.MethodologyId);
+
+                if (methodology == null)
+                {
+                    return ValidationActionResult(MethodologyDoesNotExist);
+                }
+
+                // TODO: this will want updating when methodology status is fully implemented
+                if (methodology.Status != "Draft")
```

Comment:
This isn't going to work assigning seeded methodologies which already have a status of Live.

Reply:
Good spot, inverted logic and updated tests accordingly